README.md

```diff
@@ -82,9 +82,8 @@ SQLALCHEMY_DATABASE_URI=postgresql://aaron:password@127.0.0.1:5432/resources
 ```
 8. Navigate to the cloned repo directory `cd \path\to\resources`. (see step 6 under getting started)
 9. Install virtualenv if you do not have it. `pip install virtualenv`.
-10. [Create a virtual environment](https://docs.python.org/3/library/venv.html) called venv `python -m virtualenv venv`.
-11. Activate virtual environment `source venv/bin/activate`
-12. Install the required dependencies `pip install pipenv`.
+10. [Create a virtual environment](https://docs.python.org/3/library/venv.html) called venv `python -m virtualenv venv` (If your linux machine is creating the virtualenv with python2.X you will need to run `python -m virtualenv --python=/usr/bin/python3.7 venv`. This will force the virtualenv to be created with the minimum required version of python.)
+11. Activate virtual environment `source venv/bin/activate` (This may need to be run with 'pip3' in place of 'pip'.)
```

Comment:
I think this note about pip3 was supposed to go on number 9, when the command is `pip install virtualenv` (which should, of course, be `pip3 install virtualenv`)

Reply:
Yes! I just missed that one.